lib/tasks/grade.rake

```diff
@@ -1,4 +1,120 @@
-desc "Grade project"
-task grade: :environment do
-  puts `bundle exec rspec --order default --format documentation --format j --out test_output.json --format html --out test_output.html`
+# desc "Grade project"
+# task grade: :environment do
+#   puts `bundle exec rspec --order default --format documentation --format j --out test_output.json --format html --out test_output.html`
+# end
+
+require 'json'
+require 'yaml'
+
+namespace :grade do
+
+  desc "Run tests locally"
```

Comment:
I was leaning towards not having a local-only grading option, unless we observe that it's an overwhelming amount of data to store.

Instead, I thought of at least attempting to POST every `build` to `grades.firstdraft.com`, so that we can collect interesting data over time about the difficulty of each project.

 - If the student is offline, it should fail gracefully; possibly even caching the test output for future delivery.
 - Either way, we should place a nicely formatted `test_output.html` in the `/public` folder and possibly pop it open automatically after (this might be hard on Nitrous).